Snyk.VisualStudio.Extension.Shared/Settings/SnykGeneralOptionsDialogPage.cs

```diff
@@ -84,6 +84,22 @@ private AuthenticationToken CreateAuthenticationToken(string token)
             tokenObj.TokenRefresher = RefreshToken;
             return tokenObj;
         }
+        
+        /// <summary>
+        /// Checks if the current endpoint is a fedramp endpoint
+        /// </summary>
+        /// <returns></returns>
+        public bool IsFedramp()
+        {
+            var endpoint = this.customEndpoint;
+            if (endpoint.IsNullOrEmpty())
+            {
+                return false;
+            }
+
+            var endpointUri = new Uri(endpoint);
+            return endpointUri.Host.ToLower().EndsWith("snykgov.io");
+        }
```

Comment:
(Suggestion) 
You can also do something like this:
```suggestion
        public bool IsFedramp() => > Uri.TryCreate(this.customEndpoint, UriKind.Absolute, out var uri) &&
                                 uri.Host.ToLower().EndsWith("snykgov.io");
```

It's shorter and handles any error in the URI parsing (like invalid format). I recently added code to the `CustomEndpoint` setter to avoid invalid URIs, but it's better to be safe anyway.